org.eclipse.jdt.ui/ui/org/eclipse/jdt/internal/ui/callhierarchy/FiltersDialog.java

```diff
@@ -165,18 +190,34 @@ private void updateFilterFromUI() {
         CallHierarchyUI.getDefault().setMaxCallDepth(maxCallDepth);
         CallHierarchy.getDefault().setFilters(fNames.getText());
         CallHierarchy.getDefault().setFilterEnabled(fFilterOnNames.getSelection());
-        CallHierarchy.getDefault().setFilterTestCode(fFilterTestCode.getSelection());
+
+        //try which one is selected and set the filters
+        if (fShowAll.getSelection()) {
+            CallHierarchy.getDefault().setFilter(CallHierarchy.PREF_SHOW_ALL_CODE);
+        } else if (fHideTest.getSelection()) {
+            CallHierarchy.getDefault().setFilter(CallHierarchy.PREF_HIDE_TEST_CODE);
+        } else if (fShowTest.getSelection()) {
+        	CallHierarchy.getDefault().setFilter(CallHierarchy.PREF_SHOW_TEST_CODE_ONLY);
+        }
+
     }
 
     /**
      * Updates the UI state from the given filter.
      */
     private void updateUIFromFilter() {
-      fMaxCallDepth.setText(String.valueOf(CallHierarchyUI.getDefault().getMaxCallDepth()));
-      fNames.setText(CallHierarchy.getDefault().getFilters());
-      fFilterOnNames.setSelection(CallHierarchy.getDefault().isFilterEnabled());
-      fFilterTestCode.setSelection(CallHierarchy.getDefault().isFilterTestCode());
-      updateEnabledState();
+    	  fMaxCallDepth.setText(String.valueOf(CallHierarchyUI.getDefault().getMaxCallDepth()));
+    	  fNames.setText(CallHierarchy.getDefault().getFilters());
+    	  fFilterOnNames.setSelection(CallHierarchy.getDefault().isFilterEnabled());
+
+    	   if (CallHierarchy.getDefault().isShowTestCode()) {
+    	        fShowTest.setSelection(true);
+    	    } else if (CallHierarchy.getDefault().isHideTestCode()) {
+    	        fHideTest.setSelection(true);
+    	    } else {
+    	        fShowAll.setSelection(true);
+    	    }
+    	    updateEnabledState();
```

Comment:
Fix the indentation here:
- Select the code in the body of the method
- Press `Ctrl` + `Shift` + `f`

Since the indentation changed, it looks as if you had changed the first 3 lines too, but you only changed the 4th line of the old method. This makes reviewing the changes a bit more difficult and should be avoided.

In general, if you need to change the indentation of lines you didn't edited, I like to have that done on a separate PR because then that PR is really easy to review and you make this PR (the one that makes "real" changes) also easier to review.